backend/review/views.py

```diff
@@ -23,8 +25,7 @@ def user_review(request, reviewee_id):
     if request.user.is_authenticated:
         if request.method == 'GET':
             reviewee_user = get_object_or_404(User, pk=reviewee_id)
-            # reviewee_user = User.objects.get(id=reviewee_id)
-            review_list = Review.objects.filter(reviewee=reviewee_user)
+            review_list = Review.objects.filter(reviewee=reviewee_user)[:10]
```

Comment:
I think "[:10]" will return the list of reviews in the oldest order. Please edit in the latest order.

Reply:
1. I fixed it to respond with the review list in shuffled order (not to reveal who wrote the review)
2. `[:10]` removed. (whole review list will be sent to frontend)